articles/azure-sql/database/database-import.md

```diff
@@ -139,6 +139,10 @@ az sql db import --resource-group "<resourceGroup>" --server "<server>" --name "
 > [!TIP]
 > For another script example, see [Import a database from a BACPAC file](scripts/import-from-bacpac-powershell.md).
 
+## Cancel the import request
+
+Use the procedure in the blog [How to cancel Azure SQL Database Import or Export operation](https://techcommunity.microsoft.com/t5/azure-database-support-blog/how-to-cancel-azure-sql-database-import-or-export-operation/ba-p/1935001) to cancel an ongoing import or export operation on Azure SQL Database. 
```

Comment:
We don't generally point to blogs from docs.microsoft.com. General rule is, that blogs can point to documentation, but documentation does not point to blogs, because blogs and their links can change anytime. Instead of pointing to the blog from here, could you please document a simple example of canceling an operation using REST API / PowerShell. You can then link to our REST API and Powershell documentation from here.